### PR TITLE
Listing all explicit package dependencies on the installation page…

### PR DESCRIPTION
**Pull request by rpspringuel**
…for those trying to maintain minimal TeX installations.

### CODE REVIEW

File: installation-linux.html [hunk 1]
```diff
@@ -59,7 +59,9 @@
...
+
+<p>While we recommend that you get a complete <span class="tex">T<span class="epsilon">e</span>X</span>Live installation since it will provide you with the most flexibility going forward, we understand that not everyone has the disk space or time to install the roughly 4GB of material that this requires.  If you are trying to maintain a slimmer installation, you will need to ensure that you have the following packages installed: xcolor, kvoptions, ifluatex, graphicx, luatexbase, luaotfload, luamplib, and xstring.  Most of these packages should come with the Lua<span class="tex">T<span class="epsilon">e</span>X</span> engine, but at least xcolor and xstring are known to be in other collections on some platforms.</p>
```
Comment (eroux) [inline installation-linux.html:64]: 4GB? Are you sure about that? I thought it would around 1.5GB... apart from that, it's ok to me

Comment (rpspringuel) [inline installation-linux.html:64]: The estimates come from my recent experience in reworking my Windows 10 VM.  I didn't check them against other platforms, but simply assumed that they'd be comparable.  If you think it's important, I can double check the numbers.

**Comment by rpspringuel** [comment]
If I compile the following document:

```
\documentclass{article}
\usepackage{gregoriotex}
\listfiles
\begin{document}
\end{document}
```

I get the following file list:

```
*File List*
 article.cls    2014/09/29 v1.4h Standard LaTeX document class
  size10.clo    2014/09/29 v1.4h Standard LaTeX file (size option)
gregoriotex.sty    2016/01/23 v4.1.0-beta1 GregorioTeX system.
  xcolor.sty    2007/01/21 v2.11 LaTeX color extensions (UK)
   color.cfg    2007/01/18 v1.5 color configuration of teTeX/TeXLive
  pdftex.def    2011/05/27 v0.06d Graphics/color for pdfTeX
infwarerr.sty    2010/04/08 v1.3 Providing info/warning/error messages (HO)
 ltxcmds.sty    2011/11/09 v1.22 LaTeX kernel commands for general use (HO)
kvoptions.sty    2011/06/30 v3.11 Key value format for package options (HO)
  keyval.sty    2014/10/28 v1.15 key=value parser (DPC)
kvsetkeys.sty    2012/04/25 v1.16 Key value parser (HO)
etexcmds.sty    2011/02/16 v1.5 Avoid name clashes with e-TeX commands (HO)
ifluatex.sty    2010/03/01 v1.3 Provides the ifluatex switch (HO)
graphicx.sty    2014/10/28 v1.0g Enhanced LaTeX Graphics (DPC,SPQR)
graphics.sty    2014/10/28 v1.0p Standard LaTeX Graphics (DPC,SPQR)
    trig.sty    1999/03/16 v1.09 sin cos tan (DPC)
graphics.cfg    2007/01/18 v1.5 graphics configuration of teTeX/TeXLive
luatexbase.sty    2015/10/04 v1.3 luatexbase interface to LuaTeX 
ctablestack.sty    2015/10/01 v1.0 Catcode table stable support
luaotfload.sty    2015/12/09 v2.6 OpenType layout system
luamplib.sty    2016/01/02 v2.11.2 mplib package for LuaTeX
 xstring.sty    2013/10/13  v1.7c  String manipulations (C Tellechea)
supp-pdf.mkii
 ***********
```

This indicates a few additional dependencies (mostly the dependencies of our dependencies).  Should I add them to the list of required packages?

File: installation-linux.html (hunk 1 above)
Comment (rpspringuel) [inline installation-linux.html:64]: I've double checked the installed sizes of a complete TeXLive installation (all available packages installed), and they are roughly accurate (the Mac installation was actually a bit larger, so I upped that number a bit).  Merging.

**Pull request merged**